src/Tag.xml

```diff
@@ -0,0 +1,95 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<widget id="com.mendix.widget.custom.tag.Tag" needsEntityContext="true" xmlns="http://www.mendix.com/widget/1.0/">
+    <name>Tags</name>
+    <description>Display and manipulate tag labels</description>
+    <icon>iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAAAXNSR0IArs4c6QAAAh1JREFUOBFjZCAShDY0qACVOiErZ/z37zkyHyc7uKZGNaSmJv3f////YRjI/A8SY8GpCyoB0gzU5Ly6pWX64TNnGM5fvcpgqK3NYGtiwvD3/39GJnwG+JSWqv7+/dtlFVDz379/GV5//MhQkJDACKJB4O+fP4w4XeBVVKQGVOO6pqNjys+fPxmYmZkZNJWVGXaePPlfVlwcbMC/f/+YGMEsNAKkGeg8t419fZN/fP/OwMTICMYgQ0BsEO1dVJTLzMi4C00rA4NHfr66S15e7ndgIL358uX/u69f/3/49u3/p+/f/3/58eP/b6A4SB6kDkOzY2amunNWVv5XoKKXHz/+f/X5M4ohv4DiIHmQOphmZhjDIS1NA+hsry3Tp/d//PABLAz2H9DJIMDFxcXgkp5exPDv344DM2feBAsCCbCsRUqKJtOfP5475s/vff/uHdzPML/z8fExOCcmFv9nYNh2fMGCGzDNIJrZIjZW8/+fP947Fy7sefvmDbIcmM3Hz8/gGBtbyvD//7YTixahaAYpYPnz54/vnmXLOl++egWxGSj4D6yVgUFQUJDBPiqqHBhdW8+tXHkdKoxCMQETwxb70NAKMTExBmCKA2NgWDEICgkxWAPFQfK4NMNN0vL11Vb19Kz+ANR56+XL/x+BNIgPEocrIsQQt7TUlbC0bAIZAqJBfEJ6QPIoKZFbW9sAGFjhDIyMK79evXqBGAMAuPsP791vC+0AAAAASUVORK5CYII=</icon>
+    <properties>
+        <property key="enableSuggestions" type="boolean" defaultValue="false">
+            <caption>Enable suggestions</caption>
+            <category>General</category>
+            <description>Enable usage of suggestions</description>
+        </property>
+        <property key="enableCreate" type="boolean" defaultValue="true">
+            <caption>Enable create</caption>
+            <category>General</category>
+            <description>Enables storing of anew tag</description>
+        </property>
+        <property key="lazyLoad" type="boolean" defaultValue="true">
+            <caption>Lazy load suggestions</caption>
+            <category>General</category>
+            <description>When true, suggestions are only loaded on swipe and on show, Otherwise, all tabs are loaded initially</description>
+        </property>
+        <property key="tagLimit" type="integer" defaultValue="0">
+            <caption>Tag limit</caption>
+            <category>General</category>
+            <description>Limits the number of tags that can be added, when 0 the number of tags is not limited</description>
+        </property>
+        <property key="tagStyle" type="enumeration" defaultValue="default">
```

Comment:
Can we have a style / color attribute as wel? Lets discuss with Danny

Reply:
NOT DONE